### DISCUSSION TITLE
Snake case filename option

### DISCUSSION THREAD

**Discussion by ptvirgo** [GitJournal/GitJournal#883] [Feature Requests]
What a cool app! Looks like your features are aimed at programmers & sysadmin types.

This will probably seem silly, but are you against a `snake_case_title` option for filenames?  ... because it'd make things simpler when I'm working / syncing from my laptop, where I mostly do text stuff via cli.

**Comment by justinsilvestre** [comment]
Many have requested a similar feature. I've already written some code to allow more filename formats, including snake case titles. It's linked here: https://github.com/GitJournal/GitJournal/discussions/685

Unfortunately I don't have the time to figure out how to the UI part. This should be simple for someone with a working Flutter dev environment, though. If you know of anyone, please draw their attention here 🙂
